### PR TITLE
Logging improvements flag

### PR DESCRIPTION
**Pull request by CoderDake**
Adding a flag for feature gating the new logging experience.

### CODE REVIEW

File: packages/devtools_app/lib/src/shared/feature_flags.dart [hunk 1]
```diff
@@ -70,10 +73,17 @@ abstract class FeatureFlags {
 
   /// Flag to enable DevTools extensions.
   ///
-  /// TODO(https://github.com/flutter/devtools/issues/6443): remove this flag
+  /// TODO(https://github.com/flutter/devztools/issues/6443): remove this flag
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/shared/feature_flags.dart:76]: typo

File: packages/devtools_app/lib/src/shared/feature_flags.dart [hunk 2]
```diff
@@ -70,10 +73,17 @@ abstract class FeatureFlags {
...
   /// once extension support is added in g3.
   static bool devToolsExtensions = isExternalBuild;
 
+  /// Flag to enable the new Logging experience.
+  ///
+  /// https://github.com/flutter/devtools/issues/7703
+
+  static bool loggingImprovements =
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/shared/feature_flags.dart:84]: can we just set this equal to `enableExperiments`? I don't think we need a separate flag.

File: packages/devtools_app/lib/src/shared/feature_flags.dart (hunk 1 above)
Comment (CoderDake) [reply packages/devtools_app/lib/src/shared/feature_flags.dart:76]: dang what a catch

**Review by kenzieschmoll — APPROVED** [review]
one comment but lgtm

**Pull request merged**